service/controllers/messages.go

```diff
@@ -221,6 +237,50 @@ func routes(routeObject response, w http.ResponseWriter) {
 				w.Write(jData)
 				TextToSpeech(response.Message + city + " " + state, 0)
 			}
+		} else if strings.ToLower(firstPars) == "meaning" {
```

Comment:
better if u could add other ```meaning``` conditions like '''meaning of``` and like. try this in yr next PR

Reply:
Yaa sure, i will do it in the next PR.